web-jehlomat/src/utils/payload-status.ts

```diff
@@ -0,0 +1,19 @@
+type TStatus = number | string
```

Comment:
A status is naturally a number, maybe consider keeping it a number and call `toString()` only in the `isStatusGeneralSuccess()`.